server/service/src/processors/transfer/shipment/update_outbound_shipment_status.rs

```diff
@@ -62,10 +64,14 @@ impl ShipmentTransferProcessor for UpdateOutboundShipmentStatusProcessor {
         if outbound_shipment.invoice_row.status == inbound_shipment.invoice_row.status {
             return Ok(None);
         }
+        // 6.
+        if inbound_shipment.invoice_row.status == InvoiceRowStatus::New {
```

Comment:
What happens if it's confirmed or finalised (the supplier invoice in mSupply ?)

Reply:
The OS will update to `Delivered` and `Verified` since IS will have statuses mapped correctly by then

Reply:
Sweet, just want to make sure it's been checked/tested